### PR TITLE
Application B - Voting scaffold for member contract

### PR DESCRIPTION
**Pull request by ymekuria**
**Description**

This PR adds the scaffolding for the Members smart contract of the Voting application B. This includes:

**Members Contract**
- Contract and SnarkyJS boilerplate 
- Known methods boilerplate with comments
- Location `src/examples/zkapps/voting/members.ts`

**Testing**
- Integration test script file with boilerplate and local blockchain setup
- Location `src/examples/zkapps/voting/run.ts`
- Script added to CI

### CODE REVIEW

File: src/examples/zkapps/voting/members.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  Field,
+  SmartContract,
+  state,
+  State,
+  method,
+  DeployArgs,
+  Permissions,
+} from 'snarkyjs';
+
+export class Members extends SmartContract {
```
Comment (shimkiv) [inline src/examples/zkapps/voting/members.ts:11]: Should we name it `Membership`?

File: src/examples/zkapps/voting/members.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  deploy(args: DeployArgs) {
+    super.deploy(args);
+    this.setPermissions({
+      ...Permissions.default(),
+      editState: Permissions.proofOrSignature(),
+    });
+    // TODO: Add account state initilaztion here
+  }
+
+  @method addEntry() {
```
Comment (shimkiv) [inline src/examples/zkapps/voting/members.ts:23]: We are passing an `accountId` here, don't we?

**Review by shimkiv — APPROVED** [review]
LGTM, added several comment though.

**Review by Trivo25 — APPROVED** [review]
looks good!

File: src/examples/zkapps/voting/members.ts (hunk 1 above)
Comment (ymekuria) [reply src/examples/zkapps/voting/members.ts:11]: I'm fine with changing it to Membership.

File: src/examples/zkapps/voting/membership.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    // TODO: Add account state initilaztion here
+  }
+
+  @method addEntry(accountId) {
+    // Emit event that indicates adding this item
+    // Preconditions: Restrict who can vote or who can be a candidate
+  }
+
+  @method isMember(accountId) {
+    // Verify membershiphip(voter or candidate) with the accountId via merkletree committed to by the sequence events and returns a boolean
```
Comment (shimkiv) [inline src/examples/zkapps/voting/membership.ts:29]: `membershiphip` tupo

Comment (ymekuria) [reply src/examples/zkapps/voting/membership.ts:29]: Good catch!

**Pull request merged**